components/dashboards-web-component/src/designer/DashboardDesigner.jsx

```diff
@@ -1,31 +1,30 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please revert these changes in the license header

Reply:
fixed with a2fbef4c52caad0239b8ed0f7c49136eeed8e4b2